setup.py

```diff
@@ -32,7 +32,7 @@
     python_requires='>=3.7',
     install_requires=requirements,
     extras_require={
-        'motor': ['motor>=2.0,<3.0'],
+        'motor': ['motor>=2.0,<4.0'],
```

Comment:
@lafrech should we unpin this instead?

Reply:
No, please don't unpin. This is fine as is it.